stdlib/mime/src/main/ballerina/mime/natives.bal

```diff
@@ -424,12 +432,14 @@ extern function base64Decode((string|byte[]|io:ReadableByteChannel) contentToBeD
 # + valueToBeEncoded - Content that needs to be encoded
 # + return - An encoded byte[]. In case of errors, an `error` record is returned
 public function base64EncodeBlob(byte[] valueToBeEncoded) returns byte[]|error {
-    error customErr = error("Error occurred while encoding byte[]");
-    match base64Encode(valueToBeEncoded) {
-        string returnString => return customErr;
-        byte[] returnBlob => return returnBlob;
-        io:ReadableByteChannel returnChannel => return customErr;
-        error encodeErr => return encodeErr;
+    var result = base64Encode(valueToBeEncoded);
+    if (result is byte[]) {
```

Comment:
Can we check both byte[] and error type at once as return both returns same value?
```ballerina
if (result is byte[]|error) {
  return result;
}
```